e2e/cypress/integration/ui/settings/user_mgmt.spec.ts

```diff
@@ -1,8 +1,16 @@
 describe('user management', () => {
+  const now = Cypress.moment().format('MMDDYYhhmm');
+  const typeDelay = 50;
```

Comment:
Is this common? If so, consider making it a shared constant.
If not, should add a comment why it is needed here.

Reply:
There's a comment later in the file on L38